### ISSUE TITLE
AP Physics - add space after numbering 

### ISSUE THREAD

**Issue by lwtchu** [Connexions/webview#1910]
![Screen Shot 2018-09-05 at 1.33.23 PM.png](https://images.zenhubusercontent.com/5b57310fb26dd832dfe45b43/0cd0ac2e-b690-44c8-9012-1c9fc14cd237)

Need to add a space after #26. I think when there's two paragraphs or maybe a paragraph with bold text, there isn't a space. 

https://staging.cnx.org/contents/jQSmhtXo@17.1:OF8r0KAI@17.1/Problems-Exercises

**Comment by openstaxalina** [comment]
Agree that it is better to add a space with the recipe even if this means there are two spaces in some instances.

**Comment by openstaxalina** [comment]
Space vs padding - which is better?
Decision: add **padding** to keep consistent with former decisions.
cc @Adrianltd @helenemccarron

**Comment by Adrianltd** [comment]
This was actually related to https://github.com/Connexions/webview/issues/1650
@helenemccarron do you remember why you added https://github.com/Connexions/webview/pull/1816/commits/f0cf65c92bc485e028f6ae4d56c298950c7a67c5 ?
After removing that, we have the result we need for this moment.

**Comment by helenemccarron** [comment]
Yes, there are cases when the number is followed by a divider `.`, which has the padding applied to it. If the padding is applied directly to the number, that will cause a gap between the number and the divider `.`

**Comment by Adrianltd** [comment]
So If I add the padding, I will cause regressions in those cases where the number is followed by ` . `.
I can't check spot the difference in the code, so I think I can't specify the selector in order to add padding only when there's no ` . `.
Either way there will be small issue, unless we solve it in content as was suggested in https://github.com/Connexions/webview/issues/1650

**Comment by helenemccarron** [comment]
@Adrianltd I have created this PR after talking to Alina: https://github.com/Connexions/cnx-recipes/pull/613
We decided to make sure that all the exercises would have the same format. Please remove any extra styles are now unnecessary (this should be pretty much all of the CSS targeting the numbers in exercises).

**Comment by Adrianltd** [comment]
Sorry for the mess above - I've tried to remove changes pulled from master (docker etc files), which were auto-attached because I created branch from master instead of katAW.r.r branch.
This https://github.com/Connexions/webview/pull/1934 is the correct one.
